packages/eslint-config-react-native-community/package.json

```diff
@@ -9,23 +9,23 @@
     "url": "git@github.com:facebook/react-native.git"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^1.5.0",
-    "@typescript-eslint/parser": "^1.5.0",
+    "@typescript-eslint/eslint-plugin": "^2.0.0",
+    "@typescript-eslint/parser": "^2.0.0",
     "babel-eslint": "10.0.3",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-eslint-comments": "^3.1.2",
     "eslint-plugin-flowtype": "2.50.3",
     "eslint-plugin-jest": "22.4.1",
-    "eslint-plugin-prettier": "2.6.2",
+    "eslint-plugin-prettier": "^3.0.0",
     "eslint-plugin-react": "7.16.0",
     "eslint-plugin-react-hooks": "^2.0.1",
     "eslint-plugin-react-native": "3.8.1",
-    "prettier": "1.17.0"
+    "prettier": "^1.17.0"
```

Comment:
do you think this could be bumped to `1.19.1` for typescript optional chaining support?

Reply:
is there any idea to use typescript optional chaining in react native? Sorry to beginner question